client/backwards_compatibility_test.go

```diff
@@ -278,7 +278,7 @@ func Test0Dot3RepoFormat(t *testing.T) {
 	delegations, err := repo.GetDelegationRoles()
 	require.NoError(t, err)
 	require.Len(t, delegations, 1)
-	require.Equal(t, "targets/releases", delegations[0].Name)
+	require.Equal(t, "targets/releases", string(delegations[0].Name))
```

Comment:
This is a nit, but since we're lifting types up anyway, I think this makes more sense as:

```golang
require.Equal(t, data.GUN("targets/releases"), delegations[0].Name)
```

Actually now that the datatype is `RoleName` I think `delegations[0].RoleName` would be clearer, but that could be a later PR.